NEWS.md

```diff
@@ -17,6 +17,8 @@ for human readability.
   as approximation type in `DGMulti` solvers
 - Basic support for MPI-based parallel simulations using conforming meshes of type `P4estMesh`
   in 2D and 3D
+- Basic support for MPI-based parallel simulations using non-conforming meshes of type `P4estMesh`
+  in 2D and 3D including adaptive mesh refinement
```

Comment:
```suggestion
- Initial support for MPI-based parallel simulations using non-conforming meshes of type `P4estMesh`
  in 2D and 3D including adaptive mesh refinement
```
I don't think we need two separate news entries. A user looking at this won't care we added one feature after the other but rather considers only the end result.